Packs/CommonPlaybooks/Playbooks/playbook-Calculate_Severity_By_Highest_DBotScore.yml

```diff
@@ -404,10 +493,19 @@ inputs:
       root: DBotScore
   required: false
   description: 'Array of all indicators associated with the incident. '
+  playbookInputQuery:
 outputs:
 - contextPath: Severities.DBotScoreSeverity
   description: The severity level of the incident identified and set in the Calculate Severity By Highest DBotScore playbook.
   type: string
 tests:
-- Calculate Severity - Standard - Test
-toversion: 5.9.9
\ No newline at end of file
+- Calculate Severity - Generic v2 - Test
+- Wait Until Datetime - Test
+- URL Enrichment - Generic v2 - Test
+- File Enrichment - Generic v2 - Test
+- Domain Enrichment - Generic v2 - Test
+- Account Enrichment - Generic v2.1 - Test
+- Detonate URL - Generic Test
+- Email Address Enrichment - Generic v2.1 - Test
+- Endpoint Enrichment - Generic v2.1 - Test
```

Comment:
what are all of these test playbooks that you added?

Reply:
`demisto-sdk format`